src/inmanta/protocol/common.py

```diff
@@ -856,15 +857,15 @@ def get_operation(self) -> str:
 
 
 # Util functions
-def custom_json_encoder(o: object) -> Union[ReturnTypes, util.JSONSerializable]:
+def custom_json_encoder(o: object, datetime_utc: bool = False) -> Union[ReturnTypes, util.JSONSerializable]:
```

Comment:
Why is there an option for UTC? should it not simply always be UTC?

Reply:
The method isn't used solely for the server response but also for serializing arguments to the Python client (among others). I think (but again, this is the part I find hard to reason about so I might very well be wrong) we mustn't convert those to UTC because that would be inconsistent with the REST client behavior, which doesn't convert it either. Additionally, there were test failures due to UTC conversion for the Python client though I don't really understand those failures yet.

Reply:
Aha, it's used internally as well: `BaseDocument._get_value`.